src/main/java/com/cleanroommc/groovyscript/sandbox/security/GroovySecurityManager.java

```diff
@@ -70,6 +73,10 @@ public void initDefaults() {
         banPackage("com.cleanroommc.groovyscript.registry");
         banPackage("com.cleanroommc.groovyscript.sandbox");
         banPackage("com.cleanroommc.groovyscript.server");
+        banPackage("net.prominic");
+        banMethods(IScriptReloadable.class, "onReload", "afterScriptLoad");
```

Comment:
should blacklist some methods in `ModSupport`, and maybe also `IRecipeBuilder#validate()`

![image](https://github.com/user-attachments/assets/b4c7dde8-9f7d-4102-bc90-5f3fda2c84de)